templates/rhsm-katello-reconfigure.erb

```diff
@@ -71,9 +71,14 @@ if [ -d $CA_TRUST_ANCHORS ]; then
   cp $KATELLO_CERT_DIR/$KATELLO_CERT $CA_TRUST_ANCHORS
   update-ca-trust
 
-  #restart if docker service is installed
-  service docker status >/dev/null && \
+  # restart docker if it is installed and running
+  if [ -f /usr/lib/systemd/system/docker.service ]; then
```

Comment:
This example deals with the case where the `systemctl` executable and `docker.service` unit file may or may not exist in standard locations on the system:

```sh
if systemctl list-unit-files docker.service | grep '^docker.service' >&/dev/null; then
    echo 'systemctl docker.service may be used'
elif [ -f /etc/init.d/docker ]; then
    echo 'service docker may be used'
fi
```

I am not terribly knowledgeable about `service` and where service files may exist on a system.